doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1590,6 +1590,9 @@ Binding types or coercion classes to notation scopes
    then :g:`a` of type :g:`t` in :g:`f t a` is not recognized as an argument to
    be interpreted in scope ``scope``.
 
+   In explicit casts :g:`term : coercion_class`, the :g:`term`
+   will also be interpreted in :token:`scope_name`.
```

Comment:
```suggestion
   In explicit casts :n:`@term : @coercion_class`, the :n:`@term`
   is interpreted in the :token:`scope_name` associated with :n:`@coercion_class`.
```

:n: shows syntax with hyperlinks and in green
Seemed like "also" didn't add anything.

Reply:
Thanks, I'm apparently not editing the doc often enough and forgot those things every time.

> Seemed like "also" didn't add anything.

Indeed, it's probably more confusing than useful.